src/docs/ws.md

```diff
@@ -0,0 +1,155 @@
+---
+title: " v1.0.0"
+language_tabs:
+  - javascript--node: Node.JS
+headingLevel: 3
+toc_footers:
+  - <a href="/docs">See Other REST endpoints</a>
+includes: []
+search: true
+highlight_theme: darkula
+
+---
+
+# Angular NgRx Websocket Endpoint v1.0.0
+
+> Scroll down for code samples, example headers and payloads. Select a language for code samples from the tabs above or the mobile navigation menu.
+
+Documentation for the Angular NgRx Websocket Endpoint
+
+Base URLs:
+
+* <a href="http://localhost:{port}/chat/{chatId}/send">http://localhost:{port}/chat/{chatId}/send</a>
+
+    * **chatId** - You can get your `chatId` from the get chats REST endpoint Default: 397fed73-4bcb-49ea-938b-74d4fe5ce811
+
+    * **port** -  Default: 8080
+
+* <a href="https://dev.angular-ngrx-chat.com/chat/{chatId}/send">https://dev.angular-ngrx-chat.com/chat/{chatId}/send</a>
+
+    * **chatId** - You can get your `chatId` from the get chats REST endpoint Default: 397fed73-4bcb-49ea-938b-74d4fe5ce811
+
+# Default
+
+## action.send.message
+
+<h3 id="action-send-message-publish">publish</h3>
+
+> Example headers
+
+```json
+
+{
+  "type": "object",
+  "properties": {
+    "Authorization": {
+      "description": "Bearer {{Token}}"
+    }
+  }
+}
+```
+
+> Example payload
+
+```json
+
+"Hello world!"
+```
+
+*Action to send a message.*
+
+Action to send a message.
+
+#### Headers
+
+##### Properties
+
+|Name|Type|Required|Description|
```

Comment:
I suggest getting rid of the tables.  Totally optional, but it's easier to maintain plain text docs than tables.  Here's an example: https://v1.feathers-pinia.pages.dev/guide/service-stores.html#using-definestore.

I use the following format:

```md
- `attrName {Type}` a brief and complete description. default defaultValue.  *required*
```

- `attrName {Type}` a brief and complete description. default defaultValue  **required**

Reply:
this is actually a generated document by widdershins

Reply:
I am not sure there is nothing I can do about the table